nomenclature/codes_models.py

```diff
@@ -0,0 +1,74 @@
+from re import compile, match
+from typing import Union
+from pydantic import BaseModel, validator
+
+
+TAG_PATTERN = compile("<.*>$")
```

Comment:
I tested this manually, it fails as expected on a string "a<foo>".

Reply:
Ah yes, I meant more for consistency since you end the string with a dollar sign which signals the end of a line. In principle we also would not need that but from what I've seen in json schemas, it seems to be convention to start with a caret and end with a dollar sign

Reply:
Done